week-2/Homework/extra/4-radio-stations.js

```diff
@@ -25,7 +31,32 @@
  * - Sort the stations by low - high e.g. 1,2,3,4,5
  * - Return only the frequencies that are radio stations.
  */
-// `getStations` goes here
+function getStations() {
+  let allFreq = getAllFrequencies();
+  let radioStations = [];
+  let i = 0;
+  while (allFreq[i]) {
```

Comment:
Good job! I know we probably didn't cover it by this point, but just as a point of discussion, what array method could you use to replace the `while` loop?